Addressing WGLC and YANGDOC review comments.

agva123 opened the pull request:
Addressing WGLC and YANGDOC review comments.

https://mailarchive.ietf.org/arch/msg/ccamp/liNpBNr-c4nHkWoU5v6Y9OsNcZM/
https://mailarchive.ietf.org/arch/msg/ccamp/dJU4ZXtGBwPzYQ9FRornYxXjGLs/
https://mailarchive.ietf.org/arch/msg/ccamp/CsE3EjOh5WeAw-NDhXY2yT-iaQo/
https://mailarchive.ietf.org/arch/msg/ccamp/EaueCtWSbdlW_-14SB-cc_oOYaQ/

agva123 committed:
addressing WGLC comments and YANGDOC review comments

agva123 committed:
replace tabs to align line starts

agva123 committed:
more line alignment adjustments.

italobusi commented on ietf-layer0-types.yang:
```diff
@@ -55,58 +55,58 @@ module ietf-layer0-types {
-      "The given value 'N' is used to determine the nominal
-       central frequency.
+      "The given value 'N' is used to determine the nominal central
+	   frequency.
```
```suggestion
      "The given value 'N' is used to determine the nominal central
       frequency.
```

Please fix all the tabs

italobusi commented on ietf-layer0-types.yang:
```diff
@@ -55,58 +55,58 @@ module ietf-layer0-types {
        where 193100.000 GHz (193.100000 THz) is the ITU-T 'anchor 
-         frequency' for transmission over the C band; and
+       frequency' for transmission over the C band; and
        where 'channel-spacing' is defined by the dwdm-ch-spc-type.";
```
Proposed rephrase to make it nicer to read:

```
       The nominal central frequency, 'f' is defined by:

         f = 193100.000 GHz + N x channel-spacing (measured in GHz)

       where 193100.000 GHz (193.100000 THz) is the ITU-T 'anchor 
       frequency' for transmission over the C band; and

       where  'channel-spacing' is defined by the dwdm-ch-spc-type.";
```

italobusi commented on ietf-layer0-types.yang:
```diff
@@ -500,37 +500,41 @@ module ietf-layer0-types {
-           Minimum slot width. ";
+
+           If specified, maximum slot width must be greater than or
+           equal to minimum slot width.";
```
If not specified, the maximum slot width is equal to minimum slot width.

agva123 committed:
adjusting indentations

haomianzheng committed:
Update ietf-layer0-types.yang

Solving more issues mentioned in 
https://mailarchive.ietf.org/arch/msg/ccamp/uLEFVHuigwIYetjUBes8ODOz85g/

Changes on descriptions about 'l0-label-range-info' and 'flexi-grid-label-range-info' are done.

haomianzheng committed:
Update ietf-layer0-types.yang

Change back to Rob's suggestion for description in line 484 the grouping flexi-grid-label-range-info.
